cluster-autoscaler/Dockerfile

```diff
@@ -1,19 +0,0 @@
-# Copyright 2016 The Kubernetes Authors. All rights reserved
```

Comment:
How does our release image get built? Is this going to affect that?

Reply:
We have our own dockerfiles in images/cluster-autoscaler, this doesn't affect our build process, no issue here

Reply:
we should exclusively be using the `./images/cluster-autoscaler/Dockerfile.rhel` file to build our stuff.